### DISCUSSION TITLE
Change domain to my own

### DISCUSSION THREAD

**Discussion by trkohler** [shuttle-hq/shuttle#1046] [Q&A]
Currently I have `https://*.shuttleapp.rs` for my backend, but I need to bind it to my own `https://api.my-domain.com`. How it's possible to do in shuttle?

**Comment by oddgrd** [comment]
Using custom domains is not yet publicly available, but we do have the infrastructure for it, and a few people from our community are currently trying it out. The only missing piece for a full release of this functionality is automating the renewal of certificates. We'll make sure to update this discussion when the final work is done and it can be made available to more users!
